### PR TITLE
Update files to change name from master to main

### PR DESCRIPTION
**Pull request by neha-bhargava**
Fixes # 
#3310 

**Changes proposed in this request**
Prep to rename master branch to main

- [ ] PR for one branch: https://identitydivision.visualstudio.com/IDDP/_git/MSAL.NET-OneBranch/pullrequest/5881
- [ ] PR for ext: https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/pull/181

**Testing**

**Performance impact**

### CODE REVIEW

**Review by pmaytak — APPROVED** [review]
Please update pipeline-perf-tests-automation.yaml also.

**Comment by trwalke** [comment]
Have you tried running this branch with the pr/nightly build

**Comment by neha-bhargava** [comment]
> Have you tried running this branch with the pr/nightly build

No. Any reason for trying that?

**Review by trwalke — APPROVED** [review]

File: src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
-    <PackageReleaseNotes>The changelog is available at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/changelog.txt and the roadmap at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap </PackageReleaseNotes>
+    <PackageReleaseNotes>The changelog is available at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/changelog.txt and the roadmap at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap </PackageReleaseNotes>
```
Comment (gladjohn) [inline src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj:26]: broker project does not have a mention of this so we are good there. But do we have to be consistent?

**Review by gladjohn — APPROVED** [review]

**Review by gladjohn — COMMENTED** [review]
readme_perf.txt under WebAPI needs to be updated

File: src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj (hunk 1 above)
Comment (pmaytak) [reply src/client/Microsoft.Identity.Client.Desktop/Microsoft.Identity.Client.Desktop.csproj:26]: Yea, sounds like a good idea.

**Comment by pmaytak** [comment]
Side note: GitHub automatically redirects any GitHub links that have 'master' branch in them to the default branch. As long as the builds work and such, we're good. We can update docs/wiki to point to main as we go along, imo.

**Comment by neha-bhargava** [comment]
> readme_perf.txt

Those links are stale so not changing them.

**Pull request merged**